docker/trusty/testing/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM ubuntu:trusty
 MAINTAINER Ivan Gotovchits <ivg@ieee.org>
-RUN apt-get -y update && apt-get -y install \
+
+RUN apt-get -y update
```

Comment:
nay, they are in a one `RUN` for a reason. Docker caches each `RUN` line, so it won't do `update` again, and will proceed directly to install, which will lead to missing dependencies.